README.md

```diff
@@ -0,0 +1,73 @@
+# Sturdy Lamp 💡
```

Comment:
tip: you can click on `View file` for `.md` files and see the preview of a document

![image](https://github.com/metisova/sturdy-lamp/assets/22978238/13d476c3-2ce4-4ca9-ac91-175c0434e31e)

![image](https://github.com/metisova/sturdy-lamp/assets/22978238/ec413f1a-794c-48c0-831b-bcbb30ce1e82)